src/lib/icons/IconThumbDown.svelte

```diff
@@ -1,12 +1,26 @@
-<!-- source: https://fonts.google.com/icons?selected=Material%20Icons%3Athumb_down%3A -->
+<!-- source: DFINITY foundation -->
+<script lang="ts">
+  import { DEFAULT_ICON_SIZE } from "$lib/constants/constants";
+
+  export let size = `${DEFAULT_ICON_SIZE}px`;
+</script>
+
 <svg
   xmlns="http://www.w3.org/2000/svg"
-  height="18px"
-  viewBox="0 0 24 24"
-  width="18px"
+  width={size}
```

Comment:
The previous version was `18px` and the new one is `20px` by default.
Does this look good in NNS dapp in places where we don't specify the size?
Or will we have to apply a size of `18px` in those places when we update the gix-components dep?